src/microbe_stage/FloatingChunk.cs

```diff
@@ -45,6 +45,9 @@ public class FloatingChunk : RigidBody, ISpawned, ISaveLoadedTracked
     [JsonProperty]
     private bool isParticles;
 
+    [JsonIgnore]
+    public Type SpawnerType => typeof(ChunkSpawner);
```

Comment:
Didn't you make a commit that made this SpawnerType redundant already? Moving the lists into Spawner class.